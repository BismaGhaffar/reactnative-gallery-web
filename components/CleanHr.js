import styled from 'styled-components'

const CleanHr = styled.div`
  z-index: 0;
  width: 100%;
  margin: 0 auto;
  height: 8px;
  max-width: 579.5px;
  margin-top: 28px;
  margin-bottom: 28px;
  background-image: url('/static/images/clean-hr.png');
`

export default CleanHr
