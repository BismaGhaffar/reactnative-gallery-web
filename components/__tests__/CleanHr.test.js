import React from 'react'
import renderer from 'react-test-renderer'
import CleanHr from '../CleanHr'

it('CleanHr can be created', () => {
  const comp = renderer.create(<CleanHr />)
  expect(comp).toBeDefined()
})

it('<CleanHr /> toMatchSnapshot', () => {
  const tree = renderer.create(<CleanHr />).toJSON()
  expect(tree).toMatchSnapshot()
})
