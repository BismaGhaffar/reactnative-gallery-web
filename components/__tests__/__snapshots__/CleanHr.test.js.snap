// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CleanHr /> toMatchSnapshot 1`] = `
<div
  className="CleanHr-u75eti-0 fnhkPK"
/>
`;
